source/module_pw/test/test_gnu.sh

```diff
@@ -6,21 +6,21 @@ for ((i=0;i<4;++i))
 do
 
 if ((i==0)) ;then
-make -j12 CXX=g++ DEBUG=ON > /dev/null
+make -j12 CXX=g++ DEBUG=ON > /dev/null 2>&1
```

Comment:
How about using `make -s` here?

Reply:
I think it's okay. But the current version seems better for everyone to understand. '-s' option is less known.

Reply:
> I think it's okay. But the current version seems better for everyone to understand. '-s' option is less known.

Then `--silent`

Reply:
I have committed the changes as '--silent'.